docs/setup-and-installing.md

```diff
@@ -15,10 +15,10 @@ Follow [this link](https://improbable.io/get-spatialos) to sign up for free.
-| 4 | A code editor | We recommend either [Visual Studio 2017](https://www.visualstudio.com/downloads/) or [Rider](https://www.jetbrains.com/rider/).<%(#Expandable title="How will I know when Visual Studio is installed? (Expand for image)")%>Once Visual Studio has been installed, the installer should prompt you to restart your computer.![](assets/setup/windows/visualstudio.png)<%(/Expandable)%><%(#Expandable title="How will I know when Rider is installed? (Expand for image)")%>Once Rider is installed, the installer should prompt you to click *Finish* to close Setup.![](assets/setup/windows/rider.png)<%(/Expandable)%> |
+| 1 | [Unity 2018.2.8](https://unity3d.com/get-unity/download/archive) | Make sure to download the **Installer** version, and select the following components during installation: **Linux Build Support**, **Mac Build Support**<%(#Expandable title="How will I know when Unity is installed? (Expand for image)")%>Once Unity has been installed on your machine, the Download Assistant should prompt you to click *Finish* to close the installer.![]({{assetRoot}}assets/setup/windows/unity.png)<%(/Expandable)%> |
+| 2 | [.NET Core SDK (x64)](https://www.microsoft.com/net/download/dotnet-core/2.1) | Verified with versions `2.1.3xx` and `2.1.4xx`<%(#Expandable title="How will I know when .NET Core SDK is installed? (Expand for image)")%>Once the .NET Core SDK has been installed, a window should appear stating that installation was successful and to prompt you to close the installer.![]({{assetRoot}}assets/setup/windows/dotnetcoresdk.png)<%(/Expandable)%> |
+| 3 | <a href="https://console.improbable.io/installer/download/stable/latest/win" data-track-link="SpatialOS Installer Downloaded\|product=Docs\|platform=Win" target="_blank">SpatialOS installer</a> | This installs the `spatial CLI`, the `SpatialOS Launcher`, and 32-bit and 64-bit Visual C++ Redistributables<%(#Expandable title="How will I know when SpatialOS is installed? (Expand for image)")%>Once SpatialOS has been installed, the installer should prompt you to click *Finish* to complete the installation.![]({{assetRoot}}assets/setup/windows/spatialos.png)<%(/Expandable)%> |
+| 4 | A code editor | We recommend either [Visual Studio 2017](https://www.visualstudio.com/downloads/) or [Rider](https://www.jetbrains.com/rider/).<%(#Expandable title="How will I know when Visual Studio is installed? (Expand for image)")%>Once Visual Studio has been installed, the installer should prompt you to restart your computer.![]({{assetRoot}}assets/setup/windows/visualstudio.png)<%(/Expandable)%><%(#Expandable title="How will I know when Rider is installed? (Expand for image)")%>Once Rider is installed, the installer should prompt you to click *Finish* to close Setup.![](assets/setup/windows/rider.png)<%(/Expandable)%> |
```

Comment:
There needs ot be one more {{assetRoot}} here.

Reply:
that is, `assets/setup/windows/rider.png`

Reply:
also `assets/setup/mac/visualstudio.png`